src/main/java/com/ontrack/dto/TaskDTO.java

```diff
@@ -6,12 +6,14 @@ public class TaskDTO {
 	private String name;
 	private String type;
 	private String description;
+	private String taskdate;

```

Comment:
probably should change the date from a string to something else, if you do consider using it.

Reply:
I believe Java has some kind of Date variable type, we could probably use that here.